specification/cognitiveservices/data-plane/LUIS/Authoring/preview/v3.0/LUIS-Authoring.json

```diff
@@ -8591,6 +8893,125 @@
         }
       }
     },
+    "LuisAppV2": {
+      "description": "Exported Model - An exported LUIS Application.",
+      "additionalProperties": true,
+      "type": "object",
+      "properties": {
```

Comment:
Based on the example, seems we miss `luis_schema_version` ?

Reply:
Don't see `model_features` too

Reply:
good catch !! added